### PR TITLE
FPR: update Inkscape normalization command

### PR DESCRIPTION
**Pull request by sevein**
This commit updates the command so it does not use sudo.

Connects to https://github.com/archivematica/Issues/issues/537.

### CODE REVIEW

File: src/dashboard/src/fpr/migrations/0029_update_inkscape_svg_command.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-ç
```
Comment (ross-spencer) [inline src/dashboard/src/fpr/migrations/0029_update_inkscape_svg_command.py]: `ç` -  typo (I think). Not sure it affects anything...

**Review by ross-spencer — APPROVED** [review]
One typo I think, but looks good. I appreciate you didn't split the command over multiple lines to bring it inline with 80 characters. Will we run `black` over the migrations do you think?

File: src/dashboard/src/fpr/migrations/0029_update_inkscape_svg_command.py (hunk 1 above)
Comment (sevein) [reply src/dashboard/src/fpr/migrations/0029_update_inkscape_svg_command.py]: Ha! Fixed, thanks.

**Comment by sevein** [comment]
> I appreciate you didn't split the command over multiple lines to bring it inline with 80 characters. Will we run `black` over the migrations do you think?

I guess so. I just tried to see what black would do and this is what happened:

```diff
     # The only difference with the new command is that it's not using sudo.
     new_cmd = (
-        '/usr/bin/inkscape \"%fileFullName%\" --export-plain-svg=\"%outputDirectory%%prefix%%fileName%%postfix%.svg\"'
-        '\nchmod 777 \"%outputDirectory%%prefix%%fileName%%postfix%.svg\"'
+        '/usr/bin/inkscape "%fileFullName%" --export-plain-svg="%outputDirectory%%prefix%%fileName%%postfix%.svg"'
+        '\nchmod 777 "%outputDirectory%%prefix%%fileName%%postfix%.svg"'
     )
```

Which makes sense, I was using single quotes so no need to escape double quotes.

**Pull request merged**